### DISCUSSION TITLE
Modal on top of another modal

### DISCUSSION THREAD

**Discussion by mnts-i** [mantinedev/mantine#2527] [Q&A] [answered]
I use Modals Manager to open a Context modal which inside has a link to delete an item. When the user clicks it I want to open a Confirmation Modal (via modals manager again). The problem is when the confirmation modal opens the context modal gets replaced by the confirmation modal and when the user chooses the context modal re-renders (replaces the confirmation modal)

I want to show the confirmation modal as a new modal on top of my context modal. Is there a way? Or is there any other solution like a popover message on top of the delete button?

**Comment by rtivital** [comment, marked as answer]
That is not supported, I consider multiple opened modals to be a bad UX (problems with focus trapping, escape key handling) and UI (stacked modal overlays) and it will not be supported on library level.
